proxy/src/telemetry/sensor/http.rs

```
@@ -212,7 +231,9 @@ where
                 self.handle
                     .send(|| Event::StreamRequestOpen(Arc::clone(&ctx)));
 
-                let request_open = Instant::now();
+                let RequestOpen(request_open) = req.extensions_mut()
+                    .remove::<RequestOpen>()
+                    .expect("request open timestamp should always be inserted");
```

Can we be more defensive than panicking?  Above, we are graceful to missing Server context, for instance... it seems better to fail silently (possibly logging) than to fail hard here.

---

Will do.

---

@olix0r what do you think of 55ee99876a0dc77017e884aed87cccf9fe7a1a1b?